tests/atlocal_win

```diff
@@ -167,6 +167,8 @@ if test "$GNUCOBOL_TEST_LOCAL" != "1"; then
 	done
 fi
 
+HAVE_SIGNAL_H="@HAVE_SIGNAL_H@"
```

Comment:
that should likely be a constant 1 in this file (as it is not autoconfigured but copied over)